steps/bitrise-step-stamp-appicon-with-version-number/1.0.0/step.yml

```diff
@@ -0,0 +1,63 @@
+title: Stamp AppIcon with version number
+summary: |
+  This script will use ImageMagick to stamp the version number to all icons.
+description: |
+  Stamps version "version(build number)" to the bottom of the icon.
+website: https://github.com/ollitapa/bitrise-step-stamp-appicon-with-version-number
+source_code_url: https://github.com/ollitapa/bitrise-step-stamp-appicon-with-version-number
+support_url: https://github.com/ollitapa/bitrise-step-stamp-appicon-with-version-number/issues
+published_at: !!timestamp 2018-04-04T16:50:55.395108202+03:00
+source:
+  git: https://github.com/ollitapa/bitrise-step-stamp-appicon-with-version-number.git
+  commit: 69ab704371355dc5daa4ca1ee7a8c2bf03c43fd5
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+type_tags:
+- deploy
+toolkit:
+  go:
+    package_name: github.com/ollitapa/bitrise-step-stamp-appicon-with-version-number
+deps:
+  brew:
+  - name: git
```

Comment:
do the step use `git` and `go`? if not please remove these dependencies, since them installation unnecessary increase the build time.

Reply:
Good catch! I removed git and wget. The main step script is written in go. I also updated the step-repo tag and commit id.